Strain/StrainVisualization.cpp

```diff
@@ -0,0 +1,117 @@
+#include "StrainVisualization.h"
+
+#include <iostream>
+
+
+void StrainVisualization::draw(sf::RenderTarget& target, sf::RenderStates states) const
+{
+	target.draw(_coordSystem);
+	target.draw(_torqueX);
+	target.draw(_torqueY);
+	target.draw(_torqueZ);
+	target.draw(_forceX);
+	target.draw(_forceY);
+	target.draw(_forceZ);
+}
+
+StrainVisualization::StrainVisualization(const int maxX, const int maxY, const LPCTSTR port)
+	: StrainGauge(port),
```

Comment:
**(must be changed)**: change with codestyle rules.

Reply:
Done